source/nodejs/adaptivecards-site/themes/adaptivecards/layout/archive.ejs

```diff
@@ -3,6 +3,74 @@ layout: layout
 ---
 <!-- This is the blog index page -->
 
+<script type="text/javascript">
+	function onKeyUp(event) {
+		let current = event.target;
+		// Only continue if we are focused on an article element
+		if (current.classList.contains("article")) {
+			let parentRow = current.parentElement;
+			let nextRow = parentRow.nextElementSibling;
+			let prevRow = parentRow.previousElementSibling;
+			switch(event.key) {
+				case "ArrowRight":
+					let next = current.nextElementSibling;
+					if (next != null) {
+						// Focus on next article
+						next.focus();
+					} else {
+						nextRow?.firstElementChild?.focus();
+					}
+					break;
+				case "ArrowLeft":
+					let prev = current.previousElementSibling;
+					if (prev != null) {
+						prev.focus();
+					} else {
+						prevRow?.lastElementChild?.focus();
+					}
+					break;
+				case "ArrowUp":
+					if (prevRow != null) {
+						// Special case - only one element in the row
+						if (parentRow.children.length === 1) {
```

Comment:
The up/down cases are slightly more complex so that we handle the case where the last row does not have 3 elements